clusters/app.ci/supplemental-ci-images/openshift-ocm/ocm-2.3-is.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
please add an OWNERS file with the members of your team to this directory

Reply:
Done. Took this as an opportunity to clean up our OWNERS files since I was in there.